b2luigi/__init__.py

```diff
@@ -1,9 +1,10 @@
 """Task scheduling and batch running for basf2 jobs made simple"""
-__version__ = "0.6.3"
-
 from luigi import *
 from luigi.util import inherits, copies
 
+# version mast be defined importing luigi namespace, otherwise b2luigi.__version__ gives luigi version
```

Comment:
Just some minor comment suggestions
```suggestion
# version must be defined after importing the luigi namespace, 
# otherwise the b2luigi.__version__ gets overwritten by the one from luigi
```